### PR TITLE
Enable Python in jenkins builder.

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

File: jenkins/build-opm-module.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ function build_downstreams {
     # Additional cmake parameters:
     # OPM_DATA_ROOT - passed for modules having opm-data based integration tests
     # USE_QUADMATH - used by ewoms to disable quadmath support (makes tests usable)
```
Comment (akva2) [inline jenkins/build-opm-module.sh]: please extend the comment.

Comment (joakim-hove) [inline jenkins/build-opm-module.sh]: > please extend the comment.

done

**Comment by joakim-hove** [comment]
jenkins build this with downstream please

**Comment by joakim-hove** [comment]
jenkins build this with downstream please

**Comment by akva2** [comment]
I guess the re need the lacking s brainfart taken care if. That us twice in a row now.

-------- Opprinnelig melding --------
Fra: Joakim Hove <notifications@github.com>
Dato: 28.06.2016 09:24 (GMT+01:00)
Til: OPM/opm-common <opm-common@noreply.github.com>
Ko: Arne Morten Kvarving <Arne.Morten.Kvarving@sintef.no>, Comment <comment@noreply.github.com>
Emne: Re: [OPM/opm-common] Enable Python in jenkins builder. (#151)


jenkins build this with downstream please

**Comment by joakim-hove** [comment]
> I guess the re need the lacking s brainfart taken care if. That us twice in a row now.

Crystal clear :-)

**Comment by joakim-hove** [comment]
jenkins build this with downstreams please

**Comment by joakim-hove** [comment]
jenkins build this opm-parser=860 with downstreams please

**Comment by joakim-hove** [comment]
jenkins build this opm-parser=860 with downstreams please

**Comment by joakim-hove** [comment]
Have not managed to trick Jenkins into running this: but it looks to be green from "the other side": https://github.com/OPM/opm-parser/pull/860

**Comment by akva2** [comment]
jenkins build this with downstreams opm-parser=860 please

**Comment by akva2** [comment]
just torturing jenkins to check what's going on. this looks fine to me, i would perhaps have squashed it up a bit.

**Comment by joakim-hove** [comment]
> i would perhaps have squashed it up a bit.

Two last commits squashed.

**Comment by akva2** [comment]
downstream build failures are unrelated (from the other build), caused by somebody merging changes in parser/output without checking ewoms. this can be merged as far as i am concerned.

**Comment by akva2** [comment]
(will wait for travis to not suffer my own wrath ;))

**Comment by joakim-hove** [comment]
Thank you for merging.

**Comment by joakim-hove** [comment]
> caused by somebody merging changes in parser/output without checking ewoms

That might be me; I tried integrating `ewoms` in the Travis build chain - but I lacked the required Dune foo.

**Comment by akva2** [comment]
i believe it should just be a "build_dune dune-localfunctions" in the buid-prereqs.sh file.

**Pull request merged**